### ISSUE TITLE
Empty Playlists throws Errors

### ISSUE THREAD

**Issue by tersiswilvin** [Rehike/Rehike#298]
### Rehike version

0.7

### Operating system

Windows NT 10.0.19045.3031 (Windows 10 Home)

### PHP version

8.2.0

### What's going on?

Playlists (I.E Watch Later or Liked Videos) that are empty throws a Fatal error when visited.

![image](https://github.com/Rehike/Rehike/assets/52929976/d840030e-4d06-4342-b654-a6391c1d7117)

```
Fatal error: Uncaught TypeError: Rehike\Util\ParsingUtils::getThumb(): Argument #1 ($container) must be of type object, null given, called in C:\xampp\htdocs\models\Playlist\MPlaylistHeader.php on line 27 and defined in C:\xampp\htdocs\modules\Rehike\Util\ParsingUtils.php:78 Stack trace: #0 C:\xampp\htdocs\models\Playlist\MPlaylistHeader.php(27): Rehike\Util\ParsingUtils::getThumb(NULL, 126) #1 C:\xampp\htdocs\models\Playlist\PlaylistModel.php(32): Rehike\Model\Playlist\MPlaylistHeader->__construct(Object(stdClass)) #2 C:\xampp\htdocs\controllers\playlist.php(44): Rehike\Model\Playlist\PlaylistModel::bake(Object(stdClass)) #3 [internal function]: Rehike\Controller\core\NirvanaController@anonymous->{closure}() #4 C:\xampp\htdocs\modules\Rehike\Async\Concurrency\AsyncFunction.php(133): Generator->send(Object(YukisCoffee\CoffeeRequest\Network\Response)) #5 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Promise.php(274): Rehike\Async\Concurrency\AsyncFunction->Rehike\Async\Concurrency\{closure}(Object(YukisCoffee\CoffeeRequest\Network\Response)) #6 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Promise.php(376): YukisCoffee\CoffeeRequest\Promise->resolve(Object(YukisCoffee\CoffeeRequest\Network\Response)) #7 C:\xampp\htdocs\modules\Rehike\Network.php(122): YukisCoffee\CoffeeRequest\Promise->YukisCoffee\CoffeeRequest\{closure}(Object(YukisCoffee\CoffeeRequest\Network\Response)) #8 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Promise.php(274): Rehike\Network::Rehike\{closure}(Object(YukisCoffee\CoffeeRequest\Network\Response)) #9 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Util\QueuedPromiseResolver.php(57): YukisCoffee\CoffeeRequest\Promise->resolve(Object(YukisCoffee\CoffeeRequest\Network\Response)) #10 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Loop.php(240): YukisCoffee\CoffeeRequest\Util\QueuedPromiseResolver->finish() #11 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Loop.php(230): YukisCoffee\CoffeeRequest\Loop::finishQueuedPromises() #12 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Loop.php(84): YukisCoffee\CoffeeRequest\Loop::cleanup() #13 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\CoffeeRequest.php(121): YukisCoffee\CoffeeRequest\Loop::run() #14 C:\xampp\htdocs\modules\Rehike\Network.php(214): YukisCoffee\CoffeeRequest\CoffeeRequest::run() #15 C:\xampp\htdocs\controllers\core\HitchhikerController.php(119): Rehike\Network::run() #16 C:\xampp\htdocs\modules\Rehike\ControllerV2\GetControllerInstance.php(50): Rehike\Controller\core\HitchhikerController->get(Object(Rehike\YtApp), '', Object(Rehike\ControllerV2\RequestMetadata)) #17 C:\xampp\htdocs\modules\Rehike\ControllerV2\GetControllerInstance.php(73): Rehike\ControllerV2\GetControllerInstance->_cv2WrappedCallControllerMethod('get', Array) #18 C:\xampp\htdocs\modules\Rehike\ControllerV2\Router.php(209): Rehike\ControllerV2\GetControllerInstance->__call('get', Array) #19 C:\xampp\htdocs\modules\Rehike\ControllerV2\Router.php(156): Rehike\ControllerV2\Router::pointerHandler('playlist', 'get') #20 C:\xampp\htdocs\modules\Rehike\ControllerV2\Router.php(26): Rehike\ControllerV2\Router::baseRequestMethod(Array, 'get') #21 C:\xampp\htdocs\router.php(76): Rehike\ControllerV2\Router::get(Array) #22 C:\xampp\htdocs\modules\Rehike\Boot\Bootloader.php(44): require('C:\\xampp\\htdocs...') #23 C:\xampp\htdocs\modules\Rehike\Boot\Bootloader.php(22): Rehike\Boot\Bootloader::postboot() #24 C:\xampp\htdocs\index.php(61): Rehike\Boot\Bootloader::startSession() #25 {main} thrown in C:\xampp\htdocs\modules\Rehike\Util\ParsingUtils.php on line 78
```

**Comment by tersiswilvin** [comment]
Fatal error: Uncaught TypeError: Rehike\Util\ParsingUtils::getThumb(): Argument #1 ($container) must be of type object, null given, called in C:\xampp\htdocs\models\Playlist\MPlaylistHeader.php on line 27 and defined in C:\xampp\htdocs\modules\Rehike\Util\ParsingUtils.php:78 Stack trace: #0 C:\xampp\htdocs\models\Playlist\MPlaylistHeader.php(27): Rehike\Util\ParsingUtils::getThumb(NULL, 126) #1 C:\xampp\htdocs\models\Playlist\PlaylistModel.php(32): Rehike\Model\Playlist\MPlaylistHeader->__construct(Object(stdClass)) #2 C:\xampp\htdocs\controllers\playlist.php(44): Rehike\Model\Playlist\PlaylistModel::bake(Object(stdClass)) #3 [internal function]: Rehike\Controller\core\NirvanaController@anonymous->{closure}() #4 C:\xampp\htdocs\modules\Rehike\Async\Concurrency\AsyncFunction.php(133): Generator->send(Object(YukisCoffee\CoffeeRequest\Network\Response)) #5 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Promise.php(274): Rehike\Async\Concurrency\AsyncFunction->Rehike\Async\Concurrency\{closure}(Object(YukisCoffee\CoffeeRequest\Network\Response)) #6 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Promise.php(376): YukisCoffee\CoffeeRequest\Promise->resolve(Object(YukisCoffee\CoffeeRequest\Network\Response)) #7 C:\xampp\htdocs\modules\Rehike\Network.php(122): YukisCoffee\CoffeeRequest\Promise->YukisCoffee\CoffeeRequest\{closure}(Object(YukisCoffee\CoffeeRequest\Network\Response)) #8 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Promise.php(274): Rehike\Network::Rehike\{closure}(Object(YukisCoffee\CoffeeRequest\Network\Response)) #9 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Util\QueuedPromiseResolver.php(57): YukisCoffee\CoffeeRequest\Promise->resolve(Object(YukisCoffee\CoffeeRequest\Network\Response)) #10 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Loop.php(240): YukisCoffee\CoffeeRequest\Util\QueuedPromiseResolver->finish() #11 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Loop.php(230): YukisCoffee\CoffeeRequest\Loop::finishQueuedPromises() #12 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\Loop.php(84): YukisCoffee\CoffeeRequest\Loop::cleanup() #13 C:\xampp\htdocs\modules\YukisCoffee\CoffeeRequest\CoffeeRequest.php(121): YukisCoffee\CoffeeRequest\Loop::run() #14 C:\xampp\htdocs\modules\Rehike\Network.php(214): YukisCoffee\CoffeeRequest\CoffeeRequest::run() #15 C:\xampp\htdocs\controllers\core\HitchhikerController.php(119): Rehike\Network::run() #16 C:\xampp\htdocs\modules\Rehike\ControllerV2\GetControllerInstance.php(50): Rehike\Controller\core\HitchhikerController->get(Object(Rehike\YtApp), '', Object(Rehike\ControllerV2\RequestMetadata)) #17 C:\xampp\htdocs\modules\Rehike\ControllerV2\GetControllerInstance.php(73): Rehike\ControllerV2\GetControllerInstance->_cv2WrappedCallControllerMethod('get', Array) #18 C:\xampp\htdocs\modules\Rehike\ControllerV2\Router.php(209): Rehike\ControllerV2\GetControllerInstance->__call('get', Array) #19 C:\xampp\htdocs\modules\Rehike\ControllerV2\Router.php(156): Rehike\ControllerV2\Router::pointerHandler('playlist', 'get') #20 C:\xampp\htdocs\modules\Rehike\ControllerV2\Router.php(26): Rehike\ControllerV2\Router::baseRequestMethod(Array, 'get') #21 C:\xampp\htdocs\router.php(76): Rehike\ControllerV2\Router::get(Array) #22 C:\xampp\htdocs\modules\Rehike\Boot\Bootloader.php(44): require('C:\\xampp\\htdocs...') #23 C:\xampp\htdocs\modules\Rehike\Boot\Bootloader.php(22): Rehike\Boot\Bootloader::postboot() #24 C:\xampp\htdocs\index.php(61): Rehike\Boot\Bootloader::startSession() #25 {main} thrown in C:\xampp\htdocs\modules\Rehike\Util\ParsingUtils.php on line 78

**Comment by aubymori** [comment]
Fixed in c699d75dae831e72899704160d70ea3f8e750432. Closing.
